lib/tasks/add_ost_apps.rake

```diff
@@ -11,6 +11,36 @@ namespace :accounts do
                 {:name => "OpenStax BigLearn", :prefix => "biglearn"}]
     desc "Manage applications for exercises, exchange, tutor and biglearn"
 
+    # Task specifically to create an administrative user and/or set the
+    # password for said user. This task accepts to arguments, the username
+    # and optionally a password. If the password is not supplied, the
+    # username will be used for the password.
+    task :create_admin, [:username, :password] => :environment do |t, args|
+      ActiveRecord::Base.transaction do
+        begin
+          username = args[:username]
+          password = args[:password] || args[:username]
+          admin_group = Group.find_or_create_by_name('ost_app_admin_group')
```

Comment:
Is it necessary to add the user to a group?  Is `user.is_administrator = true` not enough?